next/test/fields/array.test.ts

```diff
@@ -46,6 +46,65 @@ describe('buildFieldArray', () => {
     })
   })
 
+  it('respects x-jsf-order', () => {
+    const schema: JsfSchema = {
+      'type': 'array',
+      'x-jsf-presentation': {
+        inputType: 'group-array',
+      },
+      'items': {
+        'type': 'object',
+        'properties': {
+          first_item: { type: 'string' },
+          second_item: { type: 'string' },
+          third_item: { type: 'string' },
+        },
+        'x-jsf-order': ['second_item', 'first_item', 'third_item'],
+      },
+    }
+
+    const field = buildFieldSchema(schema, 'root', true)
+
+    expect(field).toEqual({
```

Comment:
No need for all of this, a simple assertion to the order is enough, no?

expect(fields[0].fields.map(f => f.name)).toEqual(['second_item', 'first_item', 'third_item'])

Reply:
Indeed! That makes a lot more sense.